.github/workflows/deploy_preview.yml

```diff
@@ -0,0 +1,37 @@
+name: Deploy Preview to Netlify
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy-preview:
+    name: Build and Deploy preview on Netlify
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3.2.0
+
+      - name: Install modules
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+        env:
+          REACT_APP_API_BASE_URL: ${{ secrets.REACT_APP_API_BASE_URL }}
+          REACT_APP_CLIENT_ID: ${{ secrets.REACT_APP_CLIENT_ID }}
+          REACT_APP_CLIENT_SECRET: ${{ secrets.REACT_APP_CLIENT_SECRET }}
```

Comment:
these two variables are only needed for the `nwtgck/actions-netlify@v2.0` action right? We don't use them inside the app right? If that's the case, I think we can remove them from here

```suggestion
```

Reply:
@tyrro they are used to add to the body of sign in request 🙏